src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age ? (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    ) : (
+      <>
+      </>
+    )}
```

Comment:
```suggestion
    {person.age && (
      <p className="Person__age">{`I am ${person.age}`}</p>
    )}
```
use conditional rendering instead of ternary operator if you don't have a second variant